addons/project/models/project.py

```diff
@@ -245,6 +246,14 @@ def _compute_alias_enabled(self):
         for project in self:
             project.alias_enabled = project.alias_domain and project.alias_id.alias_name
 
+    @api.depends("alias_name", "alias_domain")
+    def _compute_alias_email_formatted(self):
+        for project in self:
+            before_at_domain = ""
+            if project.alias_name:
+                before_at_domain = project.alias_name.split("@")[0]
```

Comment:
Why do you split alias_name on an "@" ? Normally alias_name is the left-part of an email, while alias_domain is the right part.

By the way, it could be interesting to have it defined in mail_alias mixin directly so that other model can use it by inheritance.

Reply:
I put up this little hack because the UI I made could lead the user to try and enter a full email address. It's just to sanitize the input. If in the image below, the prefix gets a full email, it would still work properly. 

<img width="660" alt="Capture d’écran 2020-07-02 à 11 41 02" src="https://user-images.githubusercontent.com/5216993/86343302-12199880-bc59-11ea-90f3-0e9ed71b9a58.png">

Reply:
I don't think it is a good idea ... people should not think they have to enter a full email address actually. Indeed they think they will use sales@gmail.com while their domain is company.odoo.com and would wonder why emails are not sent to gmail but to sales@company.odoo.com .

In all odoo apps aliases are left-part of emails. If there is an issue with that and we should find a generic solution but I feel like this is some side-specs not really integrated with the global mechanism of mail gateway.

Previously normally it was clear you were entering the left part because the "@company.odoo.com" was displayed as not editable content on the right of the email part. Not sure how it does look like on project currently but I think this is better compared to hacking user input.